### ISSUE TITLE
Running `npm run dev-migrations` fails on Ubuntu 16.04.1 LTS (GNU/Linux 4.4.0-34-generic x86_64)

### ISSUE THREAD

**Issue by shotaroikeda** [HackIllinois/api-2017#41]
Running `npm run dev-migrations` on a Machine running Ubuntu 16.04.1 LTS fails with this error:

```
$ ~/hackillinois/api-2017/$ npm run dev-migrations

> hackillinois-api-2017@0.0.1 dev-migrations /home/fa16-cs427/hackillinois/api-2017
> source resources/scripts/env.sh dev && ./database/flyway.sh migrate

sh: 1: source: not found

npm ERR! Linux 4.4.0-34-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "run" "dev-migrations"
npm ERR! node v4.2.6
npm ERR! npm  v3.5.2
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! hackillinois-api-2017@0.0.1 dev-migrations: `source resources/scripts/env.sh dev && ./database/flyway.sh migrate`
npm ERR! spawn ENOENT
```

In other words, `source` is not found when running `sh` rather than `bash`, although we ran the command from `bash`.

However, running `source resources/scripts/env.sh dev && ./database/flyway.sh migrate` manually in bash runs fine.

Potential fixes could be explicitly stating `npm` to use `bash` rather than `sh`.

**Comment by nmagerko** [comment]
We'll have to fix this before we release, because I have a feeling this will break during deployment to prod.

**Comment by nmagerko** [comment]
Is this still a problem? I think I fixed it in #51 but I want to make sure it works on your system.
